nomos-services/blend/src/backends/libp2p.rs

```diff
@@ -34,7 +35,9 @@ pub struct Libp2pBlendBackendSettings {
     // A key for deriving PeerId and establishing secure connections (TLS 1.3 by QUIC)
     #[serde(with = "secret_key_serde", default = "ed25519::SecretKey::generate")]
     pub node_key: ed25519::SecretKey,
-    pub conn_maintenance: ConnectionMaintenanceSettings,
+    pub peering_degree: usize,
+    pub max_peering_degree: usize,
```

Comment:
Unless we want a huge set of peering degree, we can constrain this a bit more. wdyt?

Reply:
That's actually true. In practice, I guess `u8` would be enough, but it would be safer to use `u16` because it can represent the max number of network ports that OS can use.

Reply:
I agree, `u16` sounds good.
Are we gonna change this for the crate you found?

Reply:
I updated them to `u16`: 2f4192b.
My investigation on https://docs.rs/libp2p-connection-limits/latest/libp2p_connection_limits/ is not done yet. I will open another PR.